### PR TITLE
[GEOS-10161] Smart data loader missing PostgreSQL type in DomainModelBuilder

### PR DESCRIPTION
**Pull request by taba90**
[GEOS-10161](https://osgeo-org.atlassian.net/browse/GEOS-10161)

This pr add missing type mappings between postgis and xsd in smart data loader community module.

## Checklist

> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.

**Submitting the PR does not require you to check all items, but by the time it gets merged, they should be either satisfied or not applicable.**

For all pull requests:

- [x] Confirm you have read the [contribution guidelines](https://github.com/geoserver/geoserver/blob/main/CONTRIBUTING.md) 
- [x] You have sent a Contribution Licence Agreement (CLA) as necessary (not required for small changes, e.g., fixing typos in documentation)
- [x] Make sure the first PR targets the `main` branch, eventual backports will be managed later. This can be ignored if the PR is fixing an issue that only happens in a specific branch, but not in newer ones.

The following are required only for core and extension modules (they are welcomed, but not required, for community modules):
- [ ] There is a ticket in Jira describing the issue/improvement/feature (a notable exemptions is, changes not visible to end users)
- [ ] PR for bug fixes and small new features are presented as a single commit
- [ ] Commit message must be in the form "[GEOS-XYZW] Title of the Jira ticket" (export to XML in Jira generates the message in this exact form)
- [ ] The pull request contains changes related to a single objective. If multiple focuses cannot be avoided, each one is in its own commit and has a separate ticket describing it.
- [ ] New unit tests have been added covering the changes
- [ ] This PR passes all existing unit tests (test results will be reported by Continuous Integration after opening this PR)
- [ ] This PR passes the [QA checks](https://docs.geoserver.org/latest/en/developer/qa-guide/index.html) (QA checks results will be reported by Continuous Integration after opening this PR)
- [ ] Commits changing the UI, existing user workflows, or adding new functionality, need to include documentation updates (screenshots, text)
- [ ] Commits changing the REST API, or any configuration object, should check if the REST API docs (Swagger YAML files and classic documentation) need to be updated.

### CODE REVIEW

File: src/community/smart-data-loader/src/test/resources/org/geoserver/smartdataloader/visitors/meteo-parameters-gml.xsd [hunk 1]
```diff
@@ -78,6 +78,7 @@
           <xs:element maxOccurs="1" minOccurs="0" name="name" type="xs:string" />
           <xs:element maxOccurs="1" minOccurs="0" name="surname" type="xs:string" />
           <xs:element maxOccurs="1" minOccurs="0" name="company" type="xs:string" />
+          <xs:element maxOccurs="1" minOccurs="0" name="active" type="xs:boolean" />
```
Comment (fernandor777) [inline src/community/smart-data-loader/src/test/resources/org/geoserver/smartdataloader/visitors/meteo-parameters-gml.xsd:81]: Looks like we would need at least one tested field for every supported added postgresql type.

**Review by fernandor777 — CHANGES_REQUESTED** [review]
Looks good, we need to add testing coverage for all added postgresql supported types.

File: src/community/smart-data-loader/src/test/resources/org/geoserver/smartdataloader/visitors/meteo-parameters-gml.xsd (hunk 1 above)
Comment (taba90) [reply src/community/smart-data-loader/src/test/resources/org/geoserver/smartdataloader/visitors/meteo-parameters-gml.xsd:82]: fixed

**Comment by taba90** [comment]
thanks for your review @fernandor777 feedback have been applied.

**Review by fernandor777 — APPROVED** [review]
Thanks for the update, looks good.

**Pull request merged**